frontend/appflowy_flutter/lib/workspace/presentation/home/menu/menu_user.dart

```diff
@@ -45,8 +45,29 @@ class MenuUser extends StatelessWidget {
     String iconUrl = context.read<MenuUserBloc>().state.userProfile.iconUrl;
     if (iconUrl.isEmpty) {
       iconUrl = defaultUserAvatar;
+      final String name = context.read<MenuUserBloc>().state.userProfile.name;
+      final Color nameColor = _generateRandomNameColor(name);
+      // Taking the first letters of the name components and limiting to 2 elements
+      final nameInitials = name
+          .split(' ')
+          .map((element) => element[0].toUpperCase())
```

Comment:
potential error: the element can be empty. For example: '  abc'.

Reply:
Will add trimming